cfme/tests/cloud/test_providers.py

```diff
@@ -311,3 +311,39 @@ def test_openstack_provider_has_api_version():
     pytest.sel.wait_for_ajax()
     assert pytest.sel.is_displayed(
         prov.properties_form.api_version), "API version select is not visible"
+
+
+class TestProvidersRESTAPI(object):
+    @pytest.mark.tier(3)
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.7')
+    @pytest.mark.parametrize(
+        "from_detail", [True, False],
+        ids=["from_detail", "from_collection"])
+    def test_cloud_networks_query(self, setup_a_provider, rest_api, from_detail):
+        """Tests querying cloud providers and cloud_networks collection for network info.
+
+        Metadata:
+            test_flag: rest
+        """
+        if from_detail:
+            networks = rest_api.collections.providers.get(name=setup_a_provider.name).cloud_networks
+        else:
+            networks = rest_api.collections.cloud_networks
+        assert len(networks) > 0
+        assert len(networks) == networks.subcount
```

Comment:
So, what is network.subcount == 0, perhaps we should just assert that it == the subcount?